crates/bindgen-core/build.rs

```diff
@@ -7,7 +7,7 @@ fn main() {
 }
 
 fn commit_info() {
-    if !Path::new(".git").exists() {
+    if !Path::new("../../.git").exists() {
```

Comment:
Sorry but I think that this check may not work out well. This won't play well when this is `cargo vendor`'d somewhere else, for example, since it may check for a `.git` directory that's not its own. Additionally as part of the registry this would be reaching outside of its registry location and looking for `.git`.

While this works in this git repository itself and it probably won't cause issues in other locations I'm wary to have this as it seems like an avenue to cause issues.

I don't know of a way to fix this so in retrospect I think it's fine to just print a version in the generated source code and leave off the date/commit/etc.

Reply:
That sounds fine to me. Do you still want to print date/commit info in the CLI though?

Reply:
For the CLI, yeah.

Although as I think on this more in relation to https://github.com/bytecodealliance/wit-bindgen/pull/498 instead of pulling a version from `bindgen-core` could each generator use its own version? I think we'll get to a future soon where each generator has its own independent track of versions.

Reply:
That's a good point! I will revert it back and use crate's version instead of pulling from bindgen-core.

Reply:
Okay now I reverted back to use "CARGO_PKG_VERSION" in the file header. This does not include git hash and date, but at least gives the version information.